src/Microsoft.SqlTools.ServiceLayer/TableDesigner/TableDesignerService.cs

```diff
@@ -169,9 +169,8 @@ private Task HandleGeneratePreviewReportRequest(TableInfo tableInfo, RequestCont
                 var table = this.GetTableDesigner(tableInfo);
                 var report = table.GenerateReport();
                 var generatePreviewReportResult = new GeneratePreviewReportResult();
-                // TODO - set report type by caohai
                 generatePreviewReportResult.Report = report;
-                generatePreviewReportResult.MimeType = "text/plain";
+                generatePreviewReportResult.MimeType = "text/markdown";
```

Comment:
Does this mean the report from DacFX is always generated in markdown?

Reply:
Yes, this report has been changed to use markdown format with DacFx version 160.6057.0-preview.

Reply:
Hm, well I'd definitely suggest that you consider having this be an option that people calling GenerateReport pass in. Because not everyone is going to be able to support markdown and so having a plaintext option as well is still going to be something you want IMO.

@kisantia @udeeshagautam FYI

Reply:
@Charles-Gagnon , this is not a dacfx wide thing, only table designer in dacfx is hardcoded to produce markdown report.

Reply:
@caohai I don't see an immediate need, but I guess we can add an optional parameter in the tableDesigner.GenerateScript() method to specify the output format.

Reply:
@Charles-Gagnon yeah as Alan mentioned, this is table designer specific and we are only using md for table designer preview purpose.

@alanrenmsft Sure I can add this later.